rskj-core/src/main/java/co/rsk/peg/storage/FederationStorageIndexKey.java

```diff
@@ -25,7 +25,8 @@ public enum FederationStorageIndexKey {
     NEW_FEDERATION_FORMAT_VERSION("newFederationFormatVersion"),
     OLD_FEDERATION_FORMAT_VERSION("oldFederationFormatVersion"),
     PENDING_FEDERATION_FORMAT_VERSION("pendingFederationFormatVersion"),
-    PROPOSED_FEDERATION_FORMAT_VERSION("proposedFederationFormatVersion")
+    PROPOSED_FEDERATION_FORMAT_VERSION("proposedFederationFormatVersion"),
+    SVP_FUND_TX_HASH_UNSIGNED("svp_fundTxHashUnsigned")
```

Comment:
Why do we should add an underscore in between? I prefer the current naming, and we should stick to it to be consistent.

Reply:
https://github.com/rsksmart/rskj/pull/2651#discussion_r1697486736

Reply:
Got it. Is it possible to keep the camel case at least?

Reply:
agree with natha, the underscore is disruptive haha

Reply:
Updated.